src/utils/api.js

```diff
@@ -9,4 +9,18 @@ export const getArticles = (author, topic, sort_by, order) => {
     .then((res) => {
         return res.data
     })
+}
+
+export const getArticle = (articleId) => {
+    return newsApi.get(`/articles/${articleId}`)
+    .then((res) => {
+        return res.data
+    })
```

Comment:
This api call should be in a different branch to separate out ticket 6 and ticket 5

Reply:
They are, but that branch was already in pull request because I accidentally did the tickets out of order ! Check this please
https://github.com/DanielTaylor1998/nc-news/pull/2

Reply:
No worries